src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/config/EmailConfiguration.kt

```diff
@@ -17,15 +18,7 @@ import java.util.UUID
 @Configuration
 class EmailConfiguration(
   @Value("\${notify.api.key:}") private val apiKey: String,
-  @Value("\${notify.templates.appointment-cancelled-court:}") private val appointmentCancelledCourt: String,
-  @Value("\${notify.templates.appointment-cancelled-no-court:}") private val appointmentCancelledPrisonNoCourt: String,
-  @Value("\${notify.templates.appointment-cancelled-prison:}") private val appointmentCancelledPrison: String,
-  @Value("\${notify.templates.court.released:}") private val offenderReleasedCourt: String,
-  @Value("\${notify.templates.court.transferred:}") private val offenderTransferredCourt: String,
-  @Value("\${notify.templates.prison.released:}") private val offenderReleasedPrison: String,
-  @Value("\${notify.templates.prison.released-no-court:}") private val offenderReleasedPrisonNoCourt: String,
-  @Value("\${notify.templates.prison.transferred:}") private val offenderTransferredPrison: String,
-  @Value("\${notify.templates.prison.transferred-no-court:}") private val offenderTransferredPrisonNoCourt: String,
+  @Value("\${notify.templates.court.new-booking:}") private val courtNewBooking: String,
```

Comment:
Decided to strip this back to the those that have been worked on so it is clearer going forwards.

Also I felt this would probably be the first one to work on for an end to end journey via the front end.